.github/dependabot.yaml

```diff
@@ -44,6 +44,18 @@ updates:
       - dependency-name: github.com/DataDog/datadog-agent/*
     schedule:
       interval: weekly
+  - package-ecosystem: gomod
+    directory: /pkg/obfuscate
+    labels:
+      - dependencies
+      - team/agent-apm
+      - changelog/no-changelog
+    milestone: 22
+    ignore:
+      # Ignore internal modules
+      - dependency-name: github.com/DataDog/datadog-agent/*
```

Comment:
what is the dependency bot doing?

Reply:
It ensures that the repository is using the latest version of this dependency.

Reply:
OK, nice.